packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/s3-user-input-state.ts

```diff
@@ -288,14 +288,16 @@ export class S3InputState {
   private _confirmLambdaTriggerPrefixUnique(triggerFunctionName: string, triggerPrefixList: S3TriggerPrefixType[]): boolean {
     if (this._inputPayload?.additionalTriggerFunctions) {
       for (const triggerParams of this._inputPayload.additionalTriggerFunctions) {
-        for (const configuredTriggerPrefix of triggerParams.triggerPrefix) {
-          if (
-            this.checkPrefixExists(triggerPrefixList, configuredTriggerPrefix.prefix) &&
-            triggerParams.triggerFunction !== triggerFunctionName
-          ) {
-            throw new Error(
-              `Error installing additional Lambda Trigger : trigger ${triggerParams.triggerFunction} already configured on prefix ${triggerParams.triggerPrefix}`,
-            );
+        if (triggerParams.triggerPrefix) {
```

Comment:
will this also work for empty ```triggerParams.triggerPrefix``` 
or
```suggestion
        if (_.isEmpty(triggerParams.triggerPrefix)) {
```